tensorflow/core/grappler/clusters/single_machine_test.cc

```diff
@@ -196,10 +196,19 @@ TEST_F(SingleMachineTest, GraphOptimizations) {
   TF_CHECK_OK(cluster_->Run(item.graph, item.feed, item.fetch, &metadata));
   std::set<string> cost_nodes;
   for (const auto& node : metadata.cost_graph().node()) {
+#ifdef INTEL_MKL
```

Comment:
These are nodes introduced during MKL-specific graph rewrite pass. Since TF does not expect the graph to be modified by non-standard graph rewrite passes, some unit tests fail.